frontend/freebase2wikidata.js

```diff
@@ -318,6 +367,23 @@ $(function() {
       });
     })();
 
+     // Add create Primary Sources nav list button
+    (function createPSNavList() {
```

Comment:
The button was also an "empirical" (might fail with very slow internet) way to wait for all the properties to be generated in order to avoid a partial or empty list of anchors. I will find a way to check if all properties have been generated before fetching the titles.